pom.xml

```diff
@@ -10,7 +10,7 @@
 
 	<groupId>com.github.jean-merelis</groupId>
 	<artifactId>coffeepot-br-sped-fiscal</artifactId>
-	<version>1.0.0-Beta.6</version>
+	<version>1.0.0-Beta.7</version>

```

Comment:
Acho que a versão deveria ser mantida, senão haverá um _gap_ da versão Beta.5 para Beta.7 quando eu for publicar no maven. Isto não é um problema, apenas uma observação. Vou fazer um merge mais tarde e publicar no maven. Obrigado.